bpi.syndicate.inc

```diff
@@ -174,18 +282,18 @@ function bpi_search_content($query = '', $page = 1, $filters = array(), $sort =
  *   - assets: array of links representing the images content
  */
 function bpi_search_get_items() {
-  $params = _bpi_build_query();
-  $phrase = isset($params[BPI_SEARCH_PHRASE_KEY]) ? $params[BPI_SEARCH_PHRASE_KEY] : '';
+  if ($_SERVER['REQUEST_METHOD'] != 'GET') {
```

Comment:
Function documentation needs to be updated. As the function no longer only returns an array?